src/orion/testing/__init__.py

```diff
@@ -221,6 +231,38 @@ def create_experiment(exp_config=None, trial_config=None, statuses=None):
     client.close()
 
 
+@contextmanager
+def falcon_client(exp_config=None, trial_config=None, statuses=None):
```

Comment:
Why do we need to duplicate much of the code of `create_experiment`? Could we instead re-use `create_experiment` within this function?

Reply:
done